### ISSUE TITLE
Reprinter deletes commentaries in multiline declaration

### ISSUE THREAD

**Issue by FulgurIgor** [camfort/fortran-src#115]
I have tried to reprint the following code:
```fortran
      program main
      integer, parameter :: &
      minimum = 10, & !< minimum
      maximum = 20    !< maximum
      end
```
and it produces the following output without any commentaries:
```fortran
program main
  integer, parameter :: minimum = 10, maximum = 20
end program main
```

Is it possible to save commentaries in this context?

**Comment by madgen** [comment]
Yeah, we know about this. It was mentioned in #45. Basically, this is incredibly difficult in Fortran because you can write a comment in the middle of a keyword or an identifier as well. So it is not clear where to associate the comment in the AST.

That said there is a reprinting algo in camfort/camfort that doesn't touch the untransformed bits of the AST so they remain identical to the original source code. If that' what you need, that might be your best bet.
